docker/Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7.2-alpine3.8
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PATH "/root/.poetry/bin:$PATH"
+
+RUN apk --no-cache add curl
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+
+WORKDIR /srv/app
+
+COPY ./pyproject.toml ./
+COPY ./poetry.lock ./
+
+RUN apk --no-cache add bash gcc grep libxml2-dev libxslt-dev musl-dev
```

Comment:
I think this `RUN` should be moved before the previous `COPY *` because it can be cached even if `pyproject.toml` or `poetry.lock` change; in this moment it will be re-executed for any change to these two files, but it doesn't need to. If a Python package is added that requires a new OS package, the line itself will change and purge its own cache.